staging/src/k8s.io/kubectl/pkg/cmd/create/create_role.go

```diff
@@ -116,7 +116,7 @@ var (
 func AddSpecialVerb(verb string, gr schema.GroupResource) {
 	resources, ok := specialVerbs[verb]
 	if !ok {
-		resources = make([]schema.GroupResource, 1)
+		resources = make([]schema.GroupResource, 0, 1)
```

Comment:
This is a behavioral change.  We used to have a slice with an empty `[0]` - so either this code is never called or else  someone else is observing the empty item and skipping it?

Before we merge this, we need to know why it doesn't explode with that empty item.

Can you try to track it down?

Reply:
You didn't actually answer my question.  Clearly this is WRONG today, but it's working, so SOMEWHERE we are are either ignoring empty entries in this list or somehow compensating for this bug.

What I would like to see in this PR is an explanation of why this is safe.

If I put a `panic()` inside this test, it triggers during tests.  If I grep for this function (`AddSpecialVerb()`) - there are NO callers of it.  Perhaps the bettr thing to do is delete it?  I am not super aware of kubectl internals